src/encryption.ts

```diff
@@ -0,0 +1,6 @@
+export type Eip1024EncryptedData = {
```

Comment:
A bit misleading to call this file "encryption", when really it's just a type from a deprecated EIP. Wasn't sure what else to call it though.